spyder/plugins/editor/utils/fallback.py

```diff
@@ -0,0 +1,116 @@
+
```

Comment:
Please create a module called ` spyder/plugins/editor/fallback` and move this module inside it with the name `actor`.